adal/src/main/java/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -315,12 +315,15 @@ public AuthenticationResult getAuthTokenInBackground(final AuthenticationRequest
         final Bundle bundleResult;
         if (isBrokerAccountServiceSupported()) {
             bundleResult = BrokerAccountServiceHandler.getInstance().getAuthToken(mContext, requestBundle, brokerEvent);
+            if (bundleResult == null) {
```

Comment:
i'm wondering that there will be cases when bundle result is null, it's actually indicating that interactive flow is needed.

Reply:
I tried the 2 cases:
1) no token existed, the 'bundleResult' is not NULL.
2) error returned when you redeem refresh token. the 'bundleResult' is NULL.